src/model/game/GameManager.ts

```diff
@@ -794,6 +794,12 @@ export default class GameManager {
             "https://thunderstore.io/c/gorebox/api/v1/package-listing-index/", EXCLUSIONS,
             [new StorePlatformMetadata(StorePlatform.STEAM, "2027330")], "GoreBox.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, [""]),
+        
+        new Game("TCG Card Shop Simulator", "TCG Card Shop Simulator", "TCG Card Shop Simulator",
```

Comment:
The `internalFolderName` parameter does not match the value used in `InstallationRuleApplicator.ts`, preventing the app from building.

Reply:
Should be resolved now